styles/panels.less

```diff
@@ -13,6 +13,7 @@ atom-panel {
   }
   &.bottom > *:not(:empty) {
     border-top: 1px solid @tool-panel-border-color;
+    z-index: 3;
```

Comment:
The scrollbar was going over the top border of the bottom pane, causing a weird visual break in the border. This places the bottom pane just over the scrollbar to ensure that doesn't happen.